File: server/src/main/java/de/ingrid/igeserver/exports/iso/Date.java
package de.ingrid.igeserver.exports.iso;

import javax.xml.bind.annotation.XmlElement;

public class Date {
    
    @XmlElement(name = "Date", namespace = "http://www.isotc211.org/2005/gco")
    public String date;

    public Date() {}
    
    public Date(String date) {
        this.date = date;
    }
    
}

File: server/src/main/java/de/ingrid/igeserver/exports/iso/Date.kt
package de.ingrid.igeserver.exports.iso

import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty

data class Date(
    @JacksonXmlProperty(localName = "Date") var date: String? = null,
    @JacksonXmlProperty(localName = "DateTime") var dateTime: String? = null
)
